src/style.css

```diff
@@ -0,0 +1,177 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap');
+
+:root {
+  --font-weight: 300;
+  --font: 'Poppins', sans-serif;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  margin: 0;
+  background-color: white;
+  font-family: var(--font);
+  font-weight: var(--font-weight);
+}
+
+main {
+  text-align: center;
+  padding-top: 200px;
+}
+
+ul {
+  list-style: none;
+}
+
+input {
+  width: max-content;
+  border-top-style: hidden;
+  border-right-style: hidden;
+  border-left-style: hidden;
+  border-bottom-style: hidden;
```

Comment:
- [ ] Since we are going to give the same value for all four sides. What if we simply use `border-style: hidden;` to keep the code DRY

Reply:
this has been implemented thank you